### ISSUE TITLE
No Function Url after Update

### ISSUE THREAD

**Issue by cyronis** [MSEndpointMgr/CloudLAPS#55]
Hi there,

after updating to Version 1.2.0 over the Deploy to Azure Button i dont have any Functions in the Function App anymore
The Deployment was succeeded, i also tried to deploy the zip file manually to the FunctionApp Service
Is this a known bug or did i corrupt the installation :)

**Comment by NickolajA** [comment]
1.2.0 hasn't fully been tested yet, hence why it's as a pre-release in the repo. I got side tracked at work, but I'll complete the tests shortly. However if the functions are missing, simply run the deployment again, that should hopefully fix it.

**Comment by cyronis** [comment]
I tried it but unfortunately it does not show up
But when I download it over the azure Portal  I got the content of the zip file

**Comment by johnlong-v1** [comment]
Having a similar experience to @cyronis. Doing a zip-deploy of v1.1 is a good workaround for us anyways. Zip deploy of v1.2 has no positive effect.

**Comment by JonasAxman** [comment]
Any update on this? I only read on the main page and there was no mention that it is pre release, especially since the 1.1 upgrade page is gone and no mentions of know problems there regarding this. :(

No functions in the function app anymore after upgrade, Reading up on how to do a zip deploy.

**Comment by cyronis** [comment]
Hi Jonas,

You can download the zip file for version 1.1 from the github repo

https://learn.microsoft.com/en-us/azure/app-service/deploy-zip?tabs=cli

Here is a guide to deploy the zip to the Webservice

**Comment by mark3grahams** [comment]
HI, has there been any new information regarding this? Will there be any negative repercussions if the functions are not visible?

**Comment by cyronis** [comment]
Hi @mark3grahams,
as far as my knowledge goes if the function url is not visible the password will not get sync to the web app, i would suggest to import the zip file from a lower version to get the functionality back

**Comment by jholl016** [comment]
Just ran into this today.  Thanks @cyronis for the tips on rolling back to v1.1.  That got me functional again.  After that, I did some more tinkering and eventually got 1.2 working correctly by creating a fresh ZIP file without the extra top-level folder inside the ZIP.  Did a zip deploy of my new ZIP file and it worked perfectly.  I'm guessing the extra top-level folder inside the ZIP was the problem, but I don't actually know - this is as deep as I've ever looked into function apps.
![image](https://user-images.githubusercontent.com/30354773/223217385-82991e64-433f-41cb-a4b5-02554e0a2c80.png)

**Comment by jholl016** [comment]
@mark3grahams Yeah, all client interactions are broken if you can't see the functions.  Your clients will show errors like this because the API endpoints don't exist.
![image](https://user-images.githubusercontent.com/30354773/223219249-db105476-4db0-4bea-aca4-3e41d07da22b.png)

**Comment by NickolajA** [comment]
This should be addressed now. I packaged up the new ZIP file a bit too quickly and didn't clean it up from my local build. Sorry for the inconvenience!
